.ci_support/linux_64_.yaml

```diff
@@ -22,6 +22,8 @@ python:
 - 3.8.* *_cpython
 target_platform:
 - linux-64
+tbb:
+- '2020'
```

Comment:
why previous version is requested here? I just don't understand this

Reply:
It appeared after rerender, I believe due to this PR https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1241. @isuruf, do you think it's ok?

Reply:
Yes, it's safe to ignore.

Reply:
@isuruf, thank you! Seems we are done here, so I'm going to merge the PR soon with crossed fingers 😄